<html lang="en"><head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Aerospace</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">

    <style>
      @import url("https://fonts.googleapis.com/css2?family=Kaushan+Script&display=swap");
      @import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap");
      @font-face {
        font-family: "Bariol Bold";
        src: local("Bariol Bold"), url("Bariol_Bold.woff") format("woff");
      }
      .title {
        font-family: "Bariol Bold", "Helvetica Neue", Helvetica, Arial,
          sans-serif;
      }
      body {
        background-color: #eee;
        font-family: "Lato", sans-serif;
      }
      .navbar-nav .nav-link:hover {
        color: white !important;
      }

      .table-bordered {
        border: 2px solid #000; /* 2px solid black border */
      }

      /* Style the table headers and cells */
      .table-bordered th,
      .table-bordered td {
        border: 2px solid #000; /* 2px solid black border */
        padding: 10px; /* Add padding for better readability */
      }

      .title {
        margin: 5%;
        color: #004AAD;
      }
      .container {
        width: 75%;
        /* margin: auto; */
        margin-bottom: 10%;
      }
      .card-title {
        padding: 15px;
        background-color: black;
        color: white;
        height: 50%;
        align-items: center;
      }

      .card-text {
        align-items: center;
      }
      .navbar {
        padding: 10px;
      }

      .nav-item {
        font-family: Montserrat, "Helvetica Neue", Helvetica, Arial, sans-serif;
        font-weight: 400;
        font-weight: bold;
        letter-spacing: 1px;
        background-color: #004AAD;
        z-index: 100;
      }

      .dropdown:hover .dropdown-menu {
        display: block;
      }

      .dropdown:hover .dropdown-toggle,
    .dropdown:hover .dropdown-toggle:hover {
        color: blue;
    }

      .dropdown-menu li a {
        color: white;
        background-color: black;
      }
    </style>
  </head>
  <body>
    <!-- navbar -->
    <nav class="navbar navbar-expand-lg" data-bs-theme="dark" style="background-color: #004AAD; height: 100px">
      <div class="container-fluid">
        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <a class="navbar-brand" href=""><img src="img/ugac.png" alt="" style="height: 3vmax; padding-top: 0"></a>
        <a class="navbar-brand" href=""><img src="img/enpower.jpeg" alt="" style="height: 3vmax; padding-top: 0"></a>
        <a class="navbar-brand" href="" style="
            font-family: 'Kaushan Script', 'Helvetica Neue', Helvetica, Arial,
              cursive;
            margin-right: 38vw;
          "></a>
        <div class="collapse navbar-collapse" id="navbarSupportedContent" style="margin-left: -5%">
          <ul class="navbar-nav me-auto mb-2 mb-lg-0 ml-auto">
            <li class="nav-item">
              <a class="nav-link active" aria-current="page" href="index.html">HOME</a>
            </li>
            <li class="nav-item">
              <a class="nav-link active" aria-current="page" href="index.html#about">ABOUT</a>
            </li>
            
            <li class="nav-item  ">
              <a class="nav-link active " aria-current="page" href="surp.html">
                PROJECTS
              </a>
             
            </li>
            <li class="nav-item">
              <a class="nav-link active" aria-current="page" href="https://docs.google.com/document/d/1fBzoJRVthp8Azr8a7n92JMuUeRkX08GqdRncBK0qzpM/edit?usp=sharing" target="_blank">RULEBOOK</a>
            </li>
            <li class="nav-item">
              <a class="nav-link active" aria-current="page" href="index.html#faqs">FAQ</a>
            </li>
            <li class="nav-item">
              <a class="nav-link active" aria-current="page" href="team.html">CONTACT</a>
            </li>
            
            <li class="nav-item ">
              <a class="nav-link active " href="https://forms.gle/jr3VK1DaQ4PtMe14A" target="_blank">
                APPLY NOW
              </a>
              
            </li>
          </ul>
        </div>
      </div>
    </nav>

    
    <h1 id="department-title" class="title text-center">PROJECTS IN THE KOITA CENTRE FOR DIGITAL HEALTH (KCDH) DEPARTMENT</h1>
    <div class="container">
        <div id="project-cards" class="row row-cols-1 row-cols-md-3 g-4"><div class="col">
                <div class="card h-100 shadow">
                    <h5 class="card-title text-center">Statistical models for characterizing similar cell states for community-scale single-cell data</h5>
                    <div class="card-body">
                        <div class="card-text">
                            <h6>Professor: Saket Choudhary</h6>
                            <h6>UID: DH01</h6>
                        </div>
                        <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#modal-0" style="background-color: #004AAD; border-color: #004AAD">
                            Details
                        </button>
                    </div>
                </div>
            </div><div class="col">
                <div class="card h-100 shadow">
                    <h5 class="card-title text-center">Developing statistical models for predicting celltype-specific effect of non-coding mutations</h5>
                    <div class="card-body">
                        <div class="card-text">
                            <h6>Professor: Saket Choudhary</h6>
                            <h6>UID: DH02</h6>
                        </div>
                        <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#modal-1" style="background-color: #004AAD; border-color: #004AAD">
                            Details
                        </button>
                    </div>
                </div>
            </div><div class="col">
                <div class="card h-100 shadow">
                    <h5 class="card-title text-center">Characterizing non-invasive biomarkers of fatty liver disease using comparative single-cell multiomics</h5>
                    <div class="card-body">
                        <div class="card-text">
                            <h6>Professor: Saket Choudhary</h6>
                            <h6>UID: DH03</h6>
                        </div>
                        <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#modal-2" style="background-color: #004AAD; border-color: #004AAD">
                            Details
                        </button>
                    </div>
                </div>
            </div></div>
    </div>

    <!-- Modal Template -->
    <div class="modal fade" id="projectModal" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="projectModalLabel" aria-hidden="true">
        <div class="modal-dialog modal-lg">
            <div class="modal-content">
                <div class="modal-header">
                    <h1 class="modal-title fs-5" id="projectModalLabel"></h1>
                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
                </div>
                <div class="modal-body">
                    <table class="table table-bordered">
                        <tbody id="projectDetails"></tbody>
                    </table>
                </div>
                <div class="modal-footer">
                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">
                        Close
                    </button>
                </div>
            </div>
        </div>
    </div>








    
    <div class="footer" style="width: 100%">
        <div class="container fam">
          <div class="contact" style="float: left">
            <h5 class="">Contact us <i class="fa fa-phone"></i></h5>
            <ul class="list-unstyled text-muted">
              <li class="small text-muted">
                <strong style="color: #11163d; font-size: 1.25em">Aarohi Deshpande</strong>
              </li>
              <li class="small text-muted">
                Institute Secretary Academic Affairs and Head, EnPoWER
              </li>
              <li class="small text-muted">
                Email:
                <a style="font-weight: bold; color: #151582" href="mailto:isaa.enpower.iitb@gmail.com">isaa.enpower.iitb@gmail.com</a>
              </li>
              <li class="small text-muted">
                Phone number: <strong style="color: #11163d">9137472201</strong>
              </li>
            </ul>
          </div>
          <div class="team" style="float: right">
            <h5 class="">SURP 2024</h5>
            <p class="small text-muted">
              Dive deep in the realm of Academic and Industry related research
              projects
            </p>
            Created with ❤️ by UGAC Web Team, 2024-2025
          </div>
        </div>
      </div>
      <script src="script.js"></script>
      <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ENjdO4Dr2bkBIFxQpeoTz1HIcje39Wm4jDKdf19U8gI4ddQ3GYNS7NTKfAdVQSZe" crossorigin="anonymous"></script>
    
  
  

<div class="modal fade" id="modal-0" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="modalLabel-0" aria-hidden="true">
                <div class="modal-dialog modal-lg">
                    <div class="modal-content">
                        <div class="modal-header">
                            <h1 class="modal-title fs-5" id="modalLabel-0">DH01</h1>
                            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
                        </div>
                        <div class="modal-body table-responsive">
                            <table class="table table-bordered">
                                <tbody><tr>
                                    <th>Description</th>
                                    <td>Single-cell data has seen exponential growth over the past decade. The development of molecular 'atlases' has provided unprecedented resolution for deciphering molecular mechanisms of gene regulation in the human body. However, these datasets remain largely unexplored from an ‘integration’ viewpoint - what and how molecular mechanisms get rewired in diseases. The goal of this project would be to develop large-scale statistical models to a) ‘harmonize’ cellular states across individuals, b) Decipher celltype-specific changes in molecular states across individuals, c) Identify the impact of covariates (sex, age, ethnicity) on changes in molecular states across individuals with a particular focus on diseases and ultimately d) developing a mechanistic understanding of how changes in DNA sequence drive the changes characterized as parts of aims b and c. Besides deciphering the mechanistic principle of gene regulation that will have a huge translational impact, the student will get extensive training in analyzing large-scale single-cell data and developing statistical models for interpreting genomics data applicable to the broader single-cell and bioinformatics community.</td>
                                </tr>
                                <tr>
                                    <th>Number of students</th>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>Year of study</th>
                                    <td>Students entering their 2nd year, Students entering their 3rd year, Students entering their 4th/5th year</td>
                                </tr>
                                <tr>
                                    <th>CPI</th>
                                    <td>7 and above</td>
                                </tr>
                                <tr>
                                    <th>Prerequisites</th>
                                    <td>Basic knowledge of biology recommended but not necessary; Introduction to generalized linear models recommended but not necessary</td>
                                </tr>
                                <tr>
                                    <th>Duration</th>
                                    <td>3-6 months</td>
                                </tr>
                                <tr>
                                    <th>Learning outcome</th>
                                    <td>The student will get extensive training in analyzing large-scale single-cell data and developing statistical models for interpreting genomics data applicable to the broader single-cell and bioinformatics community. Skills developed during the course of the project will be useful in both industrial and academic settings (grad schools). Besides this, you willg get hands on training on how to read, summarize,  think and solve research problems.</td>
                                </tr>
                                <tr>
                                    <th>Weekly time commitment</th>
                                    <td>30-40 hours</td>
                                </tr>
                                <tr>
                                    <th>Assignment</th>
                                    <td>https://drive.google.com/file/d/1gRwTFclvVxtNc8hj9TJ6M0aFyUC3iReh/view?usp=sharing</td>
                                </tr>
                                <tr>
                                    <th>Instructions for assignment</th>
                                    <td>Summarize the broad thematic areas of the review article. Focus on what assumptions are needed for modelling population scale data (Box 2)</td>
                                </tr>
                                  <tr>
                                    <th>Additional key points</th>
                                    <td>undefined</td>
                                </tr>
                            </tbody></table>
                        </div>
                        <div class="modal-footer">
                            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
                        </div>
                    </div>
                </div>
            </div><div class="modal fade" id="modal-1" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="modalLabel-1" aria-hidden="true">
                <div class="modal-dialog modal-lg">
                    <div class="modal-content">
                        <div class="modal-header">
                            <h1 class="modal-title fs-5" id="modalLabel-1">DH02</h1>
                            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
                        </div>
                        <div class="modal-body table-responsive">
                            <table class="table table-bordered">
                                <tbody><tr>
                                    <th>Description</th>
                                    <td>Mutations in non-coding DNA sequences are frequently associated with diseases. A current challenge in the field is to predict the effect of a non-coding mutation on gene expression given the cryptic and 
highly redundant nature of the non-coding regulatory sequences. While population genetic studies have been successful in identifying genomic loci associated with disease, they have limited interpretability for two primary reasons. First, these methods struggle at distinguishing between disease-causal and disease-associated genetic variants because of linkage disequilibrium. Second, these methods are limited in terms of their statistical power to discover associations involving rare variants.

Recent advances in deep learning have enabled integrating information from long-range interactions (up to 100kb away) in the genome to predict gene expression from DNA sequences alone. These 
methods have enabled a novel way to distill large amounts of genomic and phenotypic information into a single model. The high accuracy of these models at predicting gene expression from DNA sequence alone indicates that they are able to recognize features of the DNA sequence which are highly predictive of the phenotype (gene expression).

The current state-of-the-art (Enformer)  models to predict expression from DNA sequence alone were trained using bulk RNA-seq data which limits their ability to predict the expression state in all disease-relevant cell types in the human brain system. This project will extend the current state-of-the-arts sequence-to-expression model to  predict the impact of mutations in the non-coding genome in a celltype-specific manner by incorporating single-cell data spanning multiple species. Students will get hands-on training to apply deep learning on multiomics datasets.</td>
                                </tr>
                                <tr>
                                    <th>Number of students</th>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>Year of study</th>
                                    <td>Students entering their 2nd year, Students entering their 3rd year, Students entering their 4th/5th year</td>
                                </tr>
                                <tr>
                                    <th>CPI</th>
                                    <td>7 and above</td>
                                </tr>
                                <tr>
                                    <th>Prerequisites</th>
                                    <td>Basic knowledge of biology recommended but not necessary; Prior experience of working with deep learning models recommended but not necessary</td>
                                </tr>
                                <tr>
                                    <th>Duration</th>
                                    <td>3-6 months</td>
                                </tr>
                                <tr>
                                    <th>Learning outcome</th>
                                    <td>Students will get hands-on training to apply deep learning on multiomics datasets. Avenues for being co-authors in any publication resulting from work. Skills learned as part of the project are directly relevant to bith industry and academia (grad schools).</td>
                                </tr>
                                <tr>
                                    <th>Weekly time commitment</th>
                                    <td>30-40 hours</td>
                                </tr>
                                <tr>
                                    <th>Assignment</th>
                                    <td>https://drive.google.com/file/d/14ppymhNpz_R5pi1k_bP6HpHnkDHhOpju/view?usp=sharing</td>
                                </tr>
                                <tr>
                                    <th>Instructions for assignment</th>
                                    <td>Summarize how deep learning is currently being used for problems in genomics following the attached pdf. Write a two page summary of the entire article.</td>
                                </tr>
                                  <tr>
                                    <th>Additional key points</th>
                                    <td>undefined</td>
                                </tr>
                            </tbody></table>
                        </div>
                        <div class="modal-footer">
                            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
                        </div>
                    </div>
                </div>
            </div><div class="modal fade" id="modal-2" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="modalLabel-2" aria-hidden="true">
                <div class="modal-dialog modal-lg">
                    <div class="modal-content">
                        <div class="modal-header">
                            <h1 class="modal-title fs-5" id="modalLabel-2">DH03</h1>
                            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
                        </div>
                        <div class="modal-body table-responsive">
                            <table class="table table-bordered">
                                <tbody><tr>
                                    <th>Description</th>
                                    <td>Metabolic-associated steatotic liver disease (MASLD) or Non-alcholic fatty liver disease (NAFLD) is a chronic progressive disease arising from fat buildup in the liver leading to its inflammation, fibrosis, and cirrhosis. Liver histology is currently the reference standard for diagnosing and assessing disease progression, but biopsies are invasive and difficult to perform in steatotic patients because of health risks besides being prone to subjective variation in pathological interpretation. Despite its high prevalence, with every third adult in India suffering from MASLD}, there is a lack of robustly validated biomarkers for prognosis, diagnosis, and continuous monitoring of disease progression or regression.  With the advent of high-throughput genomics, we can characterize gene expression levels and associated regulatory states in individual celltypes across multiple donors. While a number of transcriptomic studies have been performed on MASLD patients, there have been no attempts at systematically characterizing and validating reproducible biomarkers associated with MASLD-induced pathogenicity across studies and individuals. Our project will perform large-scale analysis of single-cell RNA and open chromatin datasets to identify functional, reproducible, and validated biomarkers for MASLD.  Students will get hands-on training in genomics, statistical methods development and machine learning with a focus on transcriptomics datasets and will get to collaborate with clinicians.</td>
                                </tr>
                                <tr>
                                    <th>Number of students</th>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>Year of study</th>
                                    <td>Students entering their 2nd year, Students entering their 3rd year, Students entering their 4th/5th year</td>
                                </tr>
                                <tr>
                                    <th>CPI</th>
                                    <td>7 and above</td>
                                </tr>
                                <tr>
                                    <th>Prerequisites</th>
                                    <td>Preliminary knowledge of biology recommended but not necessary; Prior exposure to statistical models recommended but not necessary</td>
                                </tr>
                                <tr>
                                    <th>Duration</th>
                                    <td>3-6 months</td>
                                </tr>
                                <tr>
                                    <th>Learning outcome</th>
                                    <td>Students will get hands-on training in genomics, statistical methods development and machine learning with a focus on transcriptomics datasets and will get to collaborate with clinicians.</td>
                                </tr>
                                <tr>
                                    <th>Weekly time commitment</th>
                                    <td>30-40 hours</td>
                                </tr>
                                <tr>
                                    <th>Assignment</th>
                                    <td>https://drive.google.com/file/d/1K8bGxW1AgdX1WvndeJC8Ty7v0ovYltkH/view?usp=sharing</td>
                                </tr>
                                <tr>
                                    <th>Instructions for assignment</th>
                                    <td>Please summarize the attached pdf in two pages - focus on the important problems that have been solved in liver biology so far using single transcriptomics and spatial transcriptomics.</td>
                                </tr>
                                  <tr>
                                    <th>Additional key points</th>
                                    <td>undefined</td>
                                </tr>
                            </tbody></table>
                        </div>
                        <div class="modal-footer">
                            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
                        </div>
                    </div>
                </div>
            </div></body></html>